tests/datasource/fluent/test_batch_request.py

```diff
@@ -78,7 +77,7 @@ def test_batch_request_config_serialization_round_trips(
     batch_request_config: dict[str, Any] = {
         "datasource_name": datasource_name,
         "data_asset_name": data_asset_name,
-        "partitioner": PartitionerColumnValue(column_name="my_column"),
```

Comment:
unrelated to this PR, but we don't intend to support this going forward, so i replaced it with a time-based partitioner.